### ISSUE TITLE
bug: replace into occured a problem

### ISSUE THREAD

**Issue by WUWEI111222333** [datafuselabs/databend#10572]
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

8.0.26-v1.0.4-nightly-76c6334(rust-1.68.0-nightly-2023-03-06T06:37:10.700538916Z)

### What's Wrong?

the deploy mode is single 
replace into has a problem: ERROR 1105 (HY000): Code: 1068, displayText = Cannot join handle from context's runtime, cause: task 173 panicked.
![image](https://user-images.githubusercontent.com/35459358/225193566-3d521cab-f899-4065-9d4a-8bc3ea56c1f0.png)

CREATE TABLE `collector_unit_log` (
  `id` BIGINT NULL COMMENT '主键',
  `collector_unit_id` BIGINT NULL COMMENT '收运单位ID',
  `operate_user` BIGINT NULL COMMENT '操作人',
  `operate_time` TIMESTAMP NULL COMMENT '操作时间',
  `operate_type` VARCHAR NULL COMMENT '操作类型:A=暂存;B=审核通过;C=审核不通过;D=注销;E=资质过期;F=已撤回;G=待审核;',
  `gmt_create` TIMESTAMP NULL COMMENT '创建时间',
  `operate_remark` VARCHAR NULL COMMENT '操作说明',
  `gmt_modified` TIMESTAMP NULL COMMENT '修改时间'
);

replace INTO gbase.collector_unit_log on (`id`) VALUES (1, 1, 210013, '2022-02-24 10:58:16', 'G', '2022-02-24 10:58:15', NULL, '2023-02-13 14:30:21'), (2, 1, 1, '2022-02-24 11:02:40', 'B', '2022-02-24 11:02:39', NULL, '2023-02-13 14:30:21'), (3, 2, 210015, '2022-02-24 14:36:36', 'G', '2022-02-24 14:36:36', NULL, '2023-02-13 14:30:21'), (4, 2, 1, '2022-02-24 14:41:25', 'B', '2022-02-24 14:41:25', NULL, '2023-02-13 14:30:21'), (5, 3, 210016, '2022-02-24 14:46:29', 'G', '2022-02-24 14:46:29', NULL, '2023-02-13 14:30:21'), (6, 3, 1, '2022-02-24 14:46:53', 'B', '2022-02-24 14:46:53', NULL, '2023-02-13 14:30:21'), (7, 4, 210018, '2022-02-24 15:13:33', 'G', '2022-02-24 15:13:32', NULL, '2023-02-13 14:30:21'), (8, 4, 210018, '2022-02-24 15:18:17', 'F', '2022-02-24 15:18:17', NULL, '2023-02-13 14:30:21'), (9, 4, 210018, '2022-02-24 15:19:31', 'G', '2022-02-24 15:19:31', NULL, '2023-02-13 14:30:21'), (10, 4, 1, '2022-02-24 15:21:22', 'C', '2022-02-24 15:21:21', NULL, '2023-02-13 14:30:21'), (11, 1, 1, '2022-02-24 15:41:40', 'D', '2022-02-24 15:41:39', NULL, '2023-02-13 14:30:21'), (12, 5, 210019, '2022-02-24 16:02:23', 'G', '2022-02-24 16:02:23', NULL, '2023-02-13 14:30:21');

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

**Comment by wubx** [comment]
Please update you databend to databend-v1.0.16 later.  
Just only need to replace databend-query file.

**Comment by dantengsky** [comment]
hi @WUWEI111222333  thanks for opening this issue, but unfortunately I failed to reproduce it (linux box, build from source)

~~~
mysql -uroot -h127.0.0.1 -P3307
Welcome to the MySQL monitor.  Commands end with ; or \g.
Your MySQL connection id is 9
Server version: 8.0.26-v1.0.4-nightly-76c6334(rust-1.68.0-nightly-2023-03-15T03:04:53.721774986Z) 0

Copyright (c) 2000, 2023, Oracle and/or its affiliates.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.

mysql> create database gbase;
Query OK, 0 rows affected (0.05 sec)

mysql> use gbase;
Database changed
mysql> CREATE TABLE collector_unit_log (
    -> id BIGINT NULL COMMENT '主键',
    -> collector_unit_id BIGINT NULL COMMENT '收运单位ID',
    -> operate_user BIGINT NULL COMMENT '操作人',
    -> operate_time TIMESTAMP NULL COMMENT '操作时间',
    -> operate_type VARCHAR NULL COMMENT '操作类型:A=暂存;B=审核通过;C=审核不通过;D=注销;E=资质过期;F=已撤回;G=待审核;',
    -> gmt_create TIMESTAMP NULL COMMENT '创建时间',
    -> operate_remark VARCHAR NULL COMMENT '操作说明',
    -> gmt_modified TIMESTAMP NULL COMMENT '修改时间'
    -> );
Query OK, 0 rows affected (0.05 sec)

mysql> replace INTO gbase.collector_unit_log on (id) VALUES (1, 1, 210013, '2022-02-24 10:58:16', 'G', '2022-02-24 10:58:15', NULL, '2023-02-13 14:30:21'), (2, 1, 1, '2022-02-24 11:02:40', 'B', '2022-02-24 11:02:39', NULL, '2023-02-13 14:30:21'), (3, 2, 210015, '2022-02-24 14:36:36', 'G', '2022-02-24 14:36:36', NULL, '2023-02-13 14:30:21'), (4, 2, 1, '2022-02-24 14:41:25', 'B', '2022-02-24 14:41:25', NULL, '2023-02-13 14:30:21'), (5, 3, 210016, '2022-02-24 14:46:29', 'G', '2022-02-24 14:46:29', NULL, '2023-02-13 14:30:21'), (6, 3, 1, '2022-02-24 14:46:53', 'B', '2022-02-24 14:46:53', NULL, '2023-02-13 14:30:21'), (7, 4, 210018, '2022-02-24 15:13:33', 'G', '2022-02-24 15:13:32', NULL, '2023-02-13 14:30:21'), (8, 4, 210018, '2022-02-24 15:18:17', 'F', '2022-02-24 15:18:17', NULL, '2023-02-13 14:30:21'), (9, 4, 210018, '2022-02-24 15:19:31', 'G', '2022-02-24 15:19:31', NULL, '2023-02-13 14:30:21'), (10, 4, 1, '2022-02-24 15:21:22', 'C', '2022-02-24 15:21:21', NULL, '2023-02-13 14:30:21'), (11, 1, 1, '2022-02-24 15:41:40', 'D', '2022-02-24 15:41:39', NULL, '2023-02-13 14:30:21'), (12, 5, 210019, '2022-02-24 16:02:23', 'G', '2022-02-24 16:02:23', NULL, '2023-02-13 14:30:21');
Query OK, 12 rows affected (0.07 sec)

mysql> select * from collector_unit_log;
+------+-------------------+--------------+----------------------------+--------------+----------------------------+----------------+----------------------------+
| id   | collector_unit_id | operate_user | operate_time               | operate_type | gmt_create                 | operate_remark | gmt_modified               |
+------+-------------------+--------------+----------------------------+--------------+----------------------------+----------------+----------------------------+
|    1 |                 1 |       210013 | 2022-02-24 10:58:16.000000 | G            | 2022-02-24 10:58:15.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    2 |                 1 |            1 | 2022-02-24 11:02:40.000000 | B            | 2022-02-24 11:02:39.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    3 |                 2 |       210015 | 2022-02-24 14:36:36.000000 | G            | 2022-02-24 14:36:36.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    4 |                 2 |            1 | 2022-02-24 14:41:25.000000 | B            | 2022-02-24 14:41:25.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    5 |                 3 |       210016 | 2022-02-24 14:46:29.000000 | G            | 2022-02-24 14:46:29.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    6 |                 3 |            1 | 2022-02-24 14:46:53.000000 | B            | 2022-02-24 14:46:53.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    7 |                 4 |       210018 | 2022-02-24 15:13:33.000000 | G            | 2022-02-24 15:13:32.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    8 |                 4 |       210018 | 2022-02-24 15:18:17.000000 | F            | 2022-02-24 15:18:17.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|    9 |                 4 |       210018 | 2022-02-24 15:19:31.000000 | G            | 2022-02-24 15:19:31.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|   10 |                 4 |            1 | 2022-02-24 15:21:22.000000 | C            | 2022-02-24 15:21:21.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|   11 |                 1 |            1 | 2022-02-24 15:41:40.000000 | D            | 2022-02-24 15:41:39.000000 | NULL           | 2023-02-13 14:30:21.000000 |
|   12 |                 5 |       210019 | 2022-02-24 16:02:23.000000 | G            | 2022-02-24 16:02:23.000000 | NULL           | 2023-02-13 14:30:21.000000 |
+------+-------------------+--------------+----------------------------+--------------+----------------------------+----------------+----------------------------+
12 rows in set (0.01 sec)
Read 12 rows, 812.00 B in 0.001 sec., 13.53 thousand rows/sec., 893.81 KiB/sec.

mysql> select version();
+-----------------------------------------------------------------------------------+
| version()                                                                         |
+-----------------------------------------------------------------------------------+
| 8.0.26-v1.0.4-nightly-76c6334(rust-1.68.0-nightly-2023-03-15T03:04:53.721774986Z) |
+-----------------------------------------------------------------------------------+
1 row in set (0.01 sec)

~~~

**Comment by BohuTANG** [comment]
@WUWEI111222333 

From the error msg it seems the databend-query panic, can you give more databend-query logs for it?
Run as:
```
RUST_LOG=INFO ./databend-query -c <your-query.toml>
```

This will print the panic stack to the stdout.

**Comment by WUWEI111222333** [comment]
i replace the databend-query with the version 1.0.17,but the problem is the same,and the error log  as folloes:
panicked at 'assertion failed: `(left == right)`
  left: `6`,
 right: `8`', src/query/storages/fuse/src/operations/replace.rs:231:13
   0: common_tracing::panic_hook::log_panic
             at /workspace/src/common/tracing/src/panic_hook.rs:35:21
   1: common_tracing::panic_hook::set_panic_hook::{{closure}}
             at /workspace/src/common/tracing/src/panic_hook.rs:30:9
   2: <alloc::boxed::Box<F,A> as core::ops::function::Fn<Args>>::call
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/alloc/src/boxed.rs:2001:9
   3: std::panicking::rust_panic_with_hook
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:696:13
   4: std::panicking::begin_panic_handler::{{closure}}
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:583:13
   5: std::sys_common::backtrace::__rust_end_short_backtrace
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/sys_common/backtrace.rs:150:18
   6: rust_begin_unwind
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:579:5
   7: core::panicking::panic_fmt
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/panicking.rs:64:14
   8: core::panicking::assert_failed_inner
   9: core::panicking::assert_failed
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/panicking.rs:211:5
  10: common_storages_fuse::operations::replace::<impl common_storages_fuse::fuse_table::FuseTable>::build_replace_pipeline::{{closure}}
             at /workspace/src/query/storages/fuse/src/operations/replace.rs:231:13
  11: <common_storages_fuse::fuse_table::FuseTable as common_catalog::table::Table>::replace_into::{{closure}}
             at /workspace/src/query/storages/fuse/src/fuse_table.rs:501:13
  12: <core::pin::Pin<P> as core::future::future::Future>::poll
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/future/future.rs:125:9
  13: <databend_query::interpreters::interpreter_replace::ReplaceInterpreter as databend_query::interpreters::interpreter::Interpreter>::execute2::{{closure}}
             at /workspace/src/query/service/src/interpreters/interpreter_replace.rs:93:13
  14: <core::pin::Pin<P> as core::future::future::Future>::poll
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/future/future.rs:125:9
  15: databend_query::interpreters::interpreter::Interpreter::execute::{{closure}}
             at /workspace/src/query/service/src/interpreters/interpreter.rs:55:50
  16: <core::pin::Pin<P> as core::future::future::Future>::poll
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/future/future.rs:125:9
  17: databend_query::servers::mysql::mysql_interactive_worker::InteractiveWorkerBase<W>::exec_query::{{closure}}::{{closure}}::{{closure}}
             at /workspace/src/query/service/src/servers/mysql/mysql_interactive_worker.rs:386:71
  18: <tracing::instrument::Instrumented<T> as core::future::future::Future>::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tracing-0.1.37/src/instrument.rs:272:9
  19: tokio::runtime::task::core::Core<T,S>::poll::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/core.rs:223:17
  20: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/loom/std/unsafe_cell.rs:14:9
  21: tokio::runtime::task::core::Core<T,S>::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/core.rs:212:13
  22: tokio::runtime::task::harness::poll_future::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:476:19
  23: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/panic/unwind_safe.rs:271:9
  24: std::panicking::try::do_call
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:487:40
  25: std::panicking::try
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:451:19
  26: std::panic::catch_unwind
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panic.rs:140:14
  27: tokio::runtime::task::harness::poll_future
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:464:18
  28: tokio::runtime::task::harness::Harness<T,S>::poll_inner
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:198:27
  29: tokio::runtime::task::harness::Harness<T,S>::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:152:15
  30: tokio::runtime::task::raw::RawTask::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/raw.rs:200:18
  31: tokio::runtime::task::LocalNotified<S>::run
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/mod.rs:394:9
  32: tokio::runtime::scheduler::multi_thread::worker::Context::run_task::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/scheduler/multi_thread/worker.rs:464:13
  33: tokio::runtime::coop::with_budget
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/coop.rs:102:5
  34: tokio::runtime::coop::budget
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/coop.rs:68:5
  35: tokio::runtime::scheduler::multi_thread::worker::Context::run_task
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/scheduler/multi_thread/worker.rs:463:9
  36: tokio::runtime::scheduler::multi_thread::worker::Context::run
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/scheduler/multi_thread/worker.rs:426:24
  37: tokio::runtime::scheduler::multi_thread::worker::run::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/scheduler/multi_thread/worker.rs:406:17
  38: tokio::macros::scoped_tls::ScopedKey<T>::set
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/macros/scoped_tls.rs:61:9
  39: tokio::runtime::scheduler::multi_thread::worker::run
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/scheduler/multi_thread/worker.rs:403:5
  40: tokio::runtime::scheduler::multi_thread::worker::Launch::launch::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/scheduler/multi_thread/worker.rs:365:45
  41: <tokio::runtime::blocking::task::BlockingTask<T> as core::future::future::Future>::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/blocking/task.rs:42:21
  42: tokio::runtime::task::core::Core<T,S>::poll::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/core.rs:223:17
  43: tokio::loom::std::unsafe_cell::UnsafeCell<T>::with_mut
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/loom/std/unsafe_cell.rs:14:9
  44: tokio::runtime::task::core::Core<T,S>::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/core.rs:212:13
  45: tokio::runtime::task::harness::poll_future::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:476:19
  46: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/panic/unwind_safe.rs:271:9
  47: std::panicking::try::do_call
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:487:40
  48: std::panicking::try
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:451:19
  49: std::panic::catch_unwind
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panic.rs:140:14
  50: tokio::runtime::task::harness::poll_future
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:464:18
  51: tokio::runtime::task::harness::Harness<T,S>::poll_inner
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:198:27
  52: tokio::runtime::task::harness::Harness<T,S>::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/harness.rs:152:15
  53: tokio::runtime::task::raw::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/raw.rs:255:5
  54: tokio::runtime::task::raw::RawTask::poll
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/raw.rs:200:18
  55: tokio::runtime::task::UnownedTask<S>::run
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/task/mod.rs:431:9
  56: tokio::runtime::blocking::pool::Task::run
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/blocking/pool.rs:159:9
  57: tokio::runtime::blocking::pool::Inner::run
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/blocking/pool.rs:511:17
  58: tokio::runtime::blocking::pool::Spawner::spawn_thread::{{closure}}
             at /opt/rust/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.24.2/src/runtime/blocking/pool.rs:469:13
  59: std::sys_common::backtrace::__rust_begin_short_backtrace
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/sys_common/backtrace.rs:134:18
  60: std::thread::Builder::spawn_unchecked_::{{closure}}::{{closure}}
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/thread/mod.rs:560:17
  61: <core::panic::unwind_safe::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/panic/unwind_safe.rs:271:9
  62: std::panicking::try::do_call
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:487:40
  63: std::panicking::try
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panicking.rs:451:19
  64: std::panic::catch_unwind
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/panic.rs:140:14
  65: std::thread::Builder::spawn_unchecked_::{{closure}}
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/thread/mod.rs:559:30
  66: core::ops::function::FnOnce::call_once{{vtable.shim}}
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/core/src/ops/function.rs:250:5
  67: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/alloc/src/boxed.rs:1987:9
  68: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/alloc/src/boxed.rs:1987:9
  69: std::sys::unix::thread::Thread::new::thread_start
             at /rustc/39f2657d1101b50f9b71ae460b762d330cc8426b/library/std/src/sys/unix/thread.rs:108:17

**Comment by dantengsky** [comment]
got it, thanks. I will look into it

**Comment by dantengsky** [comment]
@WUWEI111222333  
hope this issue is fixed by PR #10593, which will be included in the next nightly release v1.0.20-nightly, and could be found here https://github.com/datafuselabs/databend/releases/ (tomorrow)
